imageviewer.js

```diff
@@ -4,22 +4,24 @@
     Copyright (c) 2015-2016 to Sudhanshu Yadav - ignitersworld.com , released under the MIT license.
     Demo on: http://ignitersworld.com/lab/imageViewer.html
 */
-
 /*** picture view plugin ****/
-(function ($, window, document, undefined) {
-    "use strict";
+(function(window, document, undefined) {
+    'use strict'
 
     //an empty function
-    var noop = function () {};
-
-    var $body = $('body'),
-        $window = $(window),
-        $document = $(document);
-
+    var noop = function() {}
+
+    // Event listeners
+    var eventNamespaces = {
```

Comment:
This can break in case there is multiple instances of ImageViewer. Keep the event listeners reference on `this` instead.